### ISSUE TITLE
Online Order Updates

### ISSUE THREAD

**Issue by GrantBirki** [eltana-org/eltana.com#101]
# Online Order Updates

From email:

> After inspection, you are correct. You do have storefront on there for both locations. One thing I would recommend is removing the verbiage (DoorDash) from the online ordering section. Storefront is unique from all the other 3rd parties out there, Storefront does not require your customers to have a DoorDash account. Therefore, your Postmates and Uber eats customers can order right from Storefront without needing DoorDash credentials. I'd actually recommend removing the alternative ordering options, and just having one Order Now button that goes to your Storefront page. This will ensure that every order that comes through your website is at 0% commission

**TL;DR**: Replace all "vendor" buttons with just a single "**Order Now**" button that links to our "Storefront" page
